src/components/List.js

```diff
@@ -53,14 +71,21 @@ const List = ({ firestore }) => {
 
   return (
     <>
+      {showModal && (
+        <Modal setDisplay={setModalDisplay}>
+          <h1>Warning</h1>
+          <p>Are you sure you want to delete this item?</p>
+          <button onClick={() => deleteItem()}>Yes</button>
+        </Modal>
```

Comment:
Just a thought: When we heavily start styling the application, it might be a better idea to have a `cancel` or `no` button instead of X. 
Probably we would need to create another button element.

Reply:
I was thinking we might want to label the button for deleting, 'Delete'
It would make it more clear what that button is supposed to do.

Reply:
I fixed the modal issue by moving the Modal component to the bottom of the List component instead of having it at the top.

Reply:
Works perfectly now : )

Reply:
I agree on labeling it with 'Delete'. Or we could also have an icon.